Patches/Xenn/ThingDefs_Races/Race_Xenn.xml

```diff
@@ -4,121 +4,125 @@
 	<Operation Class="PatchOperationSequence">
 		<success>Always</success>
 		<operations>
-	<!-- ========== Look for Fido ========== -->
-	<li Class="CombatExtended.PatchOperationFindMod">
+			<!-- ========== Look for Fido ========== -->
+			<li Class="CombatExtended.PatchOperationFindMod">

```

Comment:
I strongly suspect this is just a leftover from an old patch rather than FT's work, but as a general remark, `CombatExtended.PatchOperationFindMod` is no longer used in favor of the vanilla `PatchOperationFindMod`. While I won't necessitate this be changed for this (or any incoming FT patch) b/c of the tedious and pointless .xml work, newer patches should be written using the vanilla patch op.

Reply:
This particular instance was indeed left over from an old patch.

And yeah, quite a few actual FT-exclusive patches also used this PatchOp, but I'll remember to use the vanilla `PatchOperationFindMod` from now on.